modules/mono/csharp_script.cpp

```diff
@@ -1328,16 +1328,22 @@ bool CSharpScript::_update_exports() {
 		}
 
 		GDMonoClass *top = script_class;
+		List<GDMonoClass *> classes;
 
 		while (top && top != native) {
-			const Vector<GDMonoField *> &fields = top->get_all_fields();
+			classes.push_back(top);
```

Comment:
I think instead of doing this we could just change `exported_members_cache.push_back(prop_info);` below to `push_front`, and reverse the fields loop to keep the right order within the class.

Reply:
I'll try that first then :)